installer/macos/scripts/uninstall.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# This script is used to uninstall MassaStation from MacOS.
+# It will remove the following files:
+# - MassaStation App.
+# - DNSMasq configuration files.
+# - Certificates generated for MassaStation.
+# - MassaStation configuration directory.
+# - MassaStation plugins.
+
+set -e
+
+# Set MassaStation configuration directory.
+export MASSASTATION_CONFIG_DIR=/usr/local/share/massastation
+export MASSASTATION_CERT_DIR=/etc/massastation/certs
+
+# Print error message to stderr and exit with code 1.
+fatal() {
+    echo "FATAL: $1" >&2
+    exit 1
+}
+
+# Warn prints a given message in yellow.
+warn() {
+    echo -e "\033[1;33m$1\033[0m"
+}
+
+# Remove DNSMasq configuration files.
+remove_dnsmasq_configuration() {
+    sudo rm -f $(brew --prefix)/etc/dnsmasq.d/massa.conf || fatal "failed to remove DNSMasq configuration files."
+    sudo rm -f /etc/resolver/massa || fatal "failed to remove DNSMasq configuration files."
```

Comment:
just to be able to identify precisely where it fails.

```suggestion
    sudo rm -f /etc/resolver/massa || fatal "failed to remove DNSMasq resolver files."
```